config/downloads.js

```diff
@@ -10,9 +10,9 @@ module.exports = [
     downloadPath: 'apisix/2.14.1/apache-apisix-2.14.1-src',
     dockerhubPath: 'apisix',
     version: '2.14.1',
```

Comment:
Need to update this like https://github.com/apache/apisix-website/pull/1120/files